src/batch-submitter/state-batch-submitter.ts

```diff
@@ -128,11 +130,12 @@ export class StateBatchSubmitter extends BatchSubmitter {
       'appendStateBatch',
       [batch, startBlock]
     )
-    if (tx.length < this.minTxSize) {
-      this.log.info('State batch too small. Skipping batch submission...')
+    if (!this._shouldSubmitBatch(tx.length * 2)) {
```

Comment:
Is tx a string here?

Reply:
I think that its a `0x` prefixed string, to turn that into byte count isn't that divide by 2 after subtracting by 2?